packages/visualizations/src/components/Table/Cell/Cell.svelte

```diff
@@ -24,23 +26,37 @@
         text-align: right;
     }
     /* to be improved in the formatting story */
-    :global(.ods-dataviz--default td.table-data--long-text > span),
+    :global(.ods-dataviz--default td.table-data--long-text),
     :global(.ods-dataviz--default td.table-data--short-text),
     :global(.ods-dataviz--default td.table-data--url) {
         text-overflow: ellipsis;
         overflow: hidden;
-        width: max-content;
         min-width: 40px;
-        max-width: 240px;
     }
-    :global(.ods-dataviz--default td.table-data--long-text > span) {
+    :global(.ods-dataviz--default td.table-data--long-text > span),
+    :global(.ods-dataviz--default td.table-data--geo > div) {
```

Comment:
![image](https://github.com/user-attachments/assets/eacdeacc-961d-4173-a919-e40b5cbf4556)

Geo labels are no longer vertically centered

Are we sure we want to display geo labels on 3 lines?